recipe/meta.yaml

```diff
@@ -36,7 +36,10 @@ test:
 about:
   home: https://www.rabbitmq.com
   license: MPL-2.0
-  license_file: LICENSE
+  license_file: 
+    - LICENSE
```

Comment:
The current text in LICENSE is:

```
RabbitMQ server and its tier 1 (core) plugins source code is licensed under the MPL 2.0. For
the MPL2, please see LICENSE-MPL-RabbitMQ.

Some RabbitMQ server OCF files are licensed under the Apache Software License 2.0.
For the ASL2, please see LICENSE-APACHE2.

If you have any questions regarding licensing, please contact us at
rabbitmq-core@groups.vmware.com.
```

So I think it makes sense to package the actual license files as well? CC @ndevenish

Reply:
Yes, good catch, I think this was just missed before. Although I wonder if it'd just be easier to list
```
- src/LICENSE
- src/LICENSE-APACHE2
- src/LICENSE-MPL-RabbitMQ
```
rather than copying them in the build.sh

Reply:
Thanks. I went with your suggestion and it looks like it works.